tutorials/cloud-functions-sns-pubsub/index.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Connect Google Cloud Pubsub to AWS SNS topics through Cloud Functions
+description: Learn how to integrate Google Cloud Pubsub with AWS SNS using
+Google Cloud Functions.
+author: ptone
+tags: Cloud Functions, AWS, SNS, Node.js
+date_published: x
+---
+## Introduction
+
+This tutorial and sample demonstrates using [Google Cloud Functions][functions]
+to act as an integration point between the AWS Simple Notification Service
+(SNS) and Google Cloud Pubsub.  The Cloud Function is implemented in
+[Node.js][node].
+
+[functions]: https://cloud.google.com/functions
+[twilio]: https://www.twilio.com/
+[node]: https://nodejs.org/en/
+
+The sample Cloud Function is triggered by a webhook request from SNS when a
+message is published to the corresponding SNS topic. The function validates the
+sender and topic.
+
+The function then publishes the message to Cloud Pubsub - tagging the message
```

Comment:
Is this just for community? Branding guidelines say use full and parens on 1st mention.